.github/ISSUE_TEMPLATE/1-Bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: You found a bug in expresslrs. 🐞
+labels: 'bug'
+---
+
```

Comment:
Add this: 
* handset model
* opentx version (including nightly number)
* TX module hardware
* ELRS version (TX & RX MUST MATCH)
* Packet Rate
* Telemetry Ratio
* user_defines